pyrdp/player/LiveTab.py

```diff
@@ -23,7 +23,7 @@ class LiveTab(BaseTab, DirectoryObserver):
     Tab playing a live RDP connection as data is being received over the network.
     """
 
-    connectionClosed = Signal(object)
+    connectionRenameTab = Signal(object, str, bool)
```

Comment:
Why have rename and closed in the same signal? If those are 2 separate events (it seems like they are), then there should be 2 signals.

Reply:
Fixed in latest commit.